### ISSUE TITLE
Miscellaneous Learning Path Bugs

### ISSUE THREAD

**Issue by dwalizer** [NationalSecurityAgency/skills-service#2644]
- Learning Path Routes table doesn't have a title read when navigating into it
- Share Skills from this project section, Share button comes before Share With All Projects checkbox, should come after
- Screen reader for Share skills section reads the ID, which is not always accurate of the text of the Skill name
- Share skill with another project lets you select the same skill if it's already shared with All Projects - won't allow you to add it though

**Comment by rmmayo** [comment]
Learning Path Routes table still does not  have a title read when navigating into it, but this will be addressed in #2700 

:+1:
